ReactWindows/ReactNative.Shared/ReactRootView.cs

```diff
@@ -47,6 +49,17 @@ internal TouchHandler TouchHandler
         }
 
         /// <summary>
+        /// Get the initialProps
```

Comment:
Again, I followed by example. The other internal properties, referenced similarly, also have comments. e.g:

`        /// <summary>
        /// Gets the JavaScript module name.
        /// </summary>
        internal string JavaScriptModuleName
        {
            get
            {
                return _jsModuleName;
            }
        }`